automatic/gitextensions/update.ps1

```diff
@@ -1,4 +1,5 @@
 Import-Module AU
+import-module "$PSScriptRoot\..\..\scripts\au_extensions.psm1"
```

Comment:
Don't import this module manually, this is done automatically when the script `update_all.ps1` is called.
It is fine to temporarily have it when debugging the script, but should be removed once the PR gets created.

Please remove it!